main.py

```diff
@@ -1,10 +1,45 @@
+from json import load
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_data = load(file)
+
+    for player in players_data:
+        player_info = players_data[player]
+
+        race, _ = Race.objects.get_or_create(
+            name=player_info.get("race").get("name"),
+            description=player_info.get("race").get("description"),
+        ) if player_info.get("race").get("description") else None
```

Comment:
.get("race") can return none add default as dict to avoid NoneType exception fix in all places